app/models/testtrack_cli.rb

```diff
@@ -3,6 +3,10 @@
 class TesttrackCli
   include Singleton
 
+  def skip_testtrack_cli?
+    Rails.env.test? || (Rails.env.development? && project_initialized?) || ENV['SKIP_TESTTRACK_CLI'].present?
```

Comment:
nitting hard

```suggestion
    Rails.env.test? || (Rails.env.development? && project_initialized?) || ENV.key?('SKIP_TESTTRACK_CLI')
```

Reply:
should we run it through a bool-like-caster thingie so nobody gets caught out by setting `false` `f` or `0`?

Reply:
Yeah we could. It's easiest with activesupport which we should have here since Rails is a dep